classes/MenuItem.php

```diff
@@ -14,7 +14,7 @@
  * @package rainlab\pages
  * @author Alexey Bobkov, Samuel Georges
  */
-class MenuItem
+class MenuItem extends \October\Rain\Extension\Extendable
```

Comment:
@mjauvin Is there a particular reason for making this class extendable? If so, it's cleaner to use the `October\Rain\Extension\ExtendableTrait` instead.

Reply:
In order to change the MenuItem title field to an MLText formwidget, the parent "model" needs to be extendable, otherwise it will throw errors. I'll use the trait instead if it's the prefered method.